src/components/unique-token/UniqueTokenAttributes.tsx

```diff
@@ -47,18 +20,31 @@ const UniqueTokenAttributes = ({
   slug,
   traits,
 }: UniqueTokenAttributesProps) => {
-  const sortedTraits = (sortList(
-    traits,
-    'trait_type',
-    'asc'
-  ) as typeof traits).map(trait => ({
-    ...trait,
-    color,
-    disableMenu,
-    slug,
-  }));
+  const sortedTraits = useMemo(
+    () =>
+      (sortList(traits, 'trait_type', 'asc') as typeof traits)
+        .filter(trait => trait.value && trait.trait_type)
+        .sort(uniqueAssetTraitDisplayTypeCompareFunction)
```

Comment:
Should I go as far as combining `sortList` functionality with that other sort(comparator)call or is that good? I think this is more readable.

Reply:
yea, it's ok